### DISCUSSION TITLE
MultiDatasetDataLoader

### DISCUSSION THREAD

**Discussion by h-0-0** [ContinualAI/avalanche#1434] [Feature Request] [answered]
Hi, I was wondering if it would be possible to make it so the MultiDatasetDataLoader is exported? I would like to use it as I am trying out a strategy that uses a pre-training set as a buffer (where the buffer is fixed and isn't updated). I would like to be able to manually set how many samples from each dataset appear in the bath, hence why I can't use the other dataloaders in the file.

**Comment by AntonioCarta** [comment]
What kind of export do you need?

**Reply by h-0-0** [reply]
Just so I can import it from the package like so: "from avalanche.benchmarks.utils.data_loader import MultiDatasetDataLoader".

**Reply by AntonioCarta** [reply, marked as answer]
Are you using an old commit? The works correctly in the master branch.

**Reply by h-0-0** [reply]
I was using version 0.3.1, by using "pip install avalanche-lib". I've now just tried using the master branch instead and it's all working! Thanks!
